proto/sentry_protos/snuba/v1/endpoint_tags_list.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+
+//A request for "which tags are available for these projects between these dates" - used for things like autocompletion
+message TraceItemAttributesRequest {
+  RequestMeta meta = 1;
+  uint32 limit = 2;
+  uint32 offset = 3;
+  AttributeKey.Type type = 5;
+}
+
+message TraceItemAttributesResponse {
+  message Tag {
+    string name = 1;
+    AttributeKey.Type type = 2;
+  }
+  repeated Tag tags = 1;
+}
+
+message AttributeValuesRequest {
+  RequestMeta meta = 1;
+  string name = 3;
+  // a substring of the value being searched,
+  // only strict substring supported, no regex
+  string value_substring_match = 4;
+  uint32 limit = 5;
+  string page_token = 6;
+}
```

Comment:
One question I had while working on this is if this will support non attribute columns like I see segment name is available. Because autocompletion those would be nice to have. In which case it may be smart to turn the name into a attribute key

Reply:
Good question, in EAP we don't distinguish between columns which are dynamic vs directly on the table so yes it should return both